### ISSUE TITLE
`gen chromastyles` generates invalid CSS

### ISSUE THREAD

**Issue by RealOrangeOne** [gohugoio/hugo#7207]
### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version
Hugo Static Site Generator v0.69.1/extended linux/amd64 BuildDate: unknown
</pre>

### Does this issue reproduce with the latest release?

Assuming this is the latest release, yes. 

Site still builds under `0.68.3` , so I assume this is a regression?

### Issue

`hugo gen chromastyles` generates invalid CSS:

```css
/* Text */ .chroma . {  }
```

This means when minifying the file, it throws a parse error (correctly), and the site doesn't build:

```
Error: Error building site: TOCSS: failed to transform "css/highlight.css" (text/css): SCSS processing failed: file "stdin", line 82, col 11: Invalid CSS after "/* Text */": expected 1 selector or at-rule, was ".chroma . {  }" 
```

This might be an issue with something upstream? Point me in the right direction and i'll happy open an issue there!

**Comment by RealOrangeOne** [comment]
Fixed build issues by not trying to CSS the file, and my site builds now. However the CSS is still malformed, and whilst browsers are able to deal with it, it should still be fixed.

**Comment by bep** [comment]
I assume this is it? https://github.com/gohugoio/hugo/commit/102ec2da7adcc4afb7050b17989f0486f8379679

/cc @acahir

**Comment by acahir** [comment]
Yea, seeing the same thing on 0.69.2. Given that `hugo gen chroma style` is just a pass through, it was probably introduced in [cc53faa](https://github.com/alecthomas/chroma/commit/cc53faa932437033e6475ff4a248dd8ccf532375)  included in [chroma v0.7.2](https://github.com/alecthomas/chroma/releases/tag/v0.7.2).  I don't have access to standalone chroma to verify until this evening, but would just upstream to chroma.

I did test when specifying a style when outputting `hugo gen chromastyles --style=monokai`, same invalid output.

**Comment by acahir** [comment]
Was this upstreamed to Chroma? Didn't see it in that project. I can kick it over there if no one else has.

**Comment by bep** [comment]
I don't think so.

**Comment by apexskier** [comment]
Git bisecting confirms the bad css was introduced in 102ec2da7adcc4afb7050b17989f0486f8379679. I also verified that reverting the [change to `genchromastyles.go`](https://github.com/gohugoio/hugo/commit/102ec2da7adcc4afb7050b17989f0486f8379679#diff-a5206968406d1c6de057e225d4313574L71) fixes the issue.

This also reproduces with chroma directly (`chroma --html-styles --html-all-styles`). I've got a PR there (https://github.com/alecthomas/chroma/pull/358) to resolve the issue. Once that's in, updating chroma should fix the issue (or 102ec2da7adcc4afb7050b17989f0486f8379679 / https://github.com/gohugoio/hugo/pull/7168 can be reverted).
